### PR TITLE
Revert "JAMES-2709 Rely on linagora/linshare-init for LinShare LDAP i…

### PR DESCRIPTION
**Pull request by Arsnael**
…nit"

This reverts commit 4f8dc2284b385a6f3938f41642afa4eaa277918e.

### CODE REVIEW

**Comment by Arsnael** [comment]
Regarding the issue with Linshare lately... (see https://github.com/linagora/james-project/pull/2464) Hope it will help stabilize the build

File: third-party/linshare/src/test/java/org/apache/james/linshare/LDAPConfigurationPerformer.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+import java.nio.charset.StandardCharsets;
+
+import io.restassured.authentication.BasicAuthScheme;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+
+class LDAPConfigurationPerformer {
+    static void configureLdap(Linshare linshare) {
```
Comment (remk) [inline third-party/linshare/src/test/java/org/apache/james/linshare/LDAPConfigurationPerformer.java]: This method is a little long. You coud factorize the different building blocks ?

**Comment by Arsnael** [comment]
> I don't agree. We should ask linshare team some help.

Well sure... 

We checked with Benoit and they didn't change those tags since we use them though so... Not sure why it worked before and we are having issues now. Might be just sometimes we just run out of resources so it crashes...

File: third-party/linshare/src/test/java/org/apache/james/linshare/LDAPConfigurationPerformer.java (hunk 1 above)
Comment (Arsnael) [reply third-party/linshare/src/test/java/org/apache/james/linshare/LDAPConfigurationPerformer.java]: Will take this remark into consideration if the PR stops being blocked. Thanks :)

**Comment by trantienduchn** [comment]
was :green_apple: once. I just re-run the build to make sure it fix this problem (maybe need more few times). test this please

**Comment by trantienduchn** [comment]
was passed maven test twice. test this please

**Comment by Arsnael** [comment]
I don't believe the issue would be solved like this anymore, let's just close it.

**Pull request closed without merging**